server/chatbotDashboard.mst

```diff
@@ -201,6 +203,7 @@
                                 <td>{{notes}}</td>
                                 <td>{{createdAt}}</td>
                                 <td>{{updatedAt}}</td>
+                                <td>{{buttonBatteryLevel}}</td>
```

Comment:
This variable should match the name in the `viewParams` object. So either this should be `{{batteryLevel}}` or it should be `buttonBatteryLevel` in `viewParams`.